lib/__tests__/cookieJar.spec.ts

```diff
@@ -1489,25 +1496,27 @@ describe('Synchronous API on async CookieJar', () => {
 })
 
 describe('validation errors invoke callbacks', () => {
-  it('getCookies', () => {
+  it('getCookies', (done) => {
     const invalidUrl = {}
     const cookieJar = new CookieJar()
     // @ts-expect-error deliberately trigger validation error
     void cookieJar.getCookies(invalidUrl, (err) => {
       expect(err).toMatchObject({
         message: '`url` argument is not a string or URL.',
       })
+      done()
```

Comment:
good catch. of course, i fix a bug where callbacks were not being invoked and then wrote tests that fail to signal the test is done in the callback 🤦